src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the checklist, the `get()` method in the service layer should return the entity, not an `Optional`. However, the dao layer should return `Optional`. Please make sure that the dao layer's `get()` method returns `Optional<MovieSession>` and the service layer handles the `Optional` accordingly.